src/components/structures/LeftPanel2.tsx

```diff
@@ -205,14 +206,19 @@ export default class LeftPanel2 extends React.Component<IProps, IState> {
             "mx_LeftPanel2_minimized": this.props.isMinimized,
         });
 
+        const className = classnames(
```

Comment:
to avoid confusion with the others, we should probably call this `roomListClasses` or something.